public/urls.json

```diff
@@ -1,8 +1,18 @@
 {
-  "edd-ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
-  "edd-ui-eligibility": "https://edd.ca.gov/Unemployment/Eligibility.htm",
+  "ca-gov": "https://www.ca.gov/",
+  "edd-about-accessibility": "https://www.edd.ca.gov/About_EDD/Accessibility.htm",
+  "edd-about-conditions-of-use": "https://www.edd.ca.gov/About_EDD/Conditions_Of_Use.htm",
+  "edd-about-contact": "https://www.edd.ca.gov/About_EDD/Contact_EDD.htm",
+  "edd-about-privacy-policy": "https://www.edd.ca.gov/About_EDD/Privacy_Policy.htm",
+  "edd-ca-gov": "https://edd.ca.gov",
   "edd-covid": "https://www.edd.ca.gov/about_edd/coronavirus-2019/benefit-year-end.htm",
```

Comment:
following the convention of my other links to `https://www.edd.ca.gov/About_EDD/[...]` this would get renamed to `edd-about-covid` but I didn't want to step on any toes

Reply:
Feel free to rename!

Reply:
```suggestion
  "edd-about-covid": "https://www.edd.ca.gov/about_edd/coronavirus-2019/benefit-year-end.htm",
```